client/templates/website-template/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "website-template",
+  "license": "agpl-3.0-or-later",
+  "private": true,
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```

At least for the template

---

I'm not sure what a `0.0.0` dependency does with NPM. I don't like playing with fire.

---

It just means unversioned - the code is always considered to have 'changed' e.g. if you push it to npm it will override the existing code and won't complain, even if you didn't increment it in the meantime. I'd rather use a convention like that than have someone accidentally commit to a 1.0.0 release when making a new package.

---

TIL